SU-HKKU/cardinal_service_profile
# CSD-387 Updated spotlight list and filter styles

# READY FOR REVIEW

# Summary
- Updated spotlight list and filter styles

# Needed By (Date)
- 09/01

# Urgency
- medium

# Steps to Test

1. Pull this PR
2. Clear cache
3. View Our Stories page `http://cardinalservice.ace-gryphon.loc/our-stories`
- Images should be 210px x 210px
- Image should display on mobile
- Filter should have `Search By` title (this should appear on Resources as well)
- Filter should have the same sandstone/tan background as other filter pages

# Associated Issues and/or People
- [CSD-387](https://stanfordits.atlassian.net/browse/CSD-387)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
